docs/content/preview/explore/joins/joins-ysql.md

```diff
@@ -0,0 +1,233 @@
+---
```

Comment:
This page should go either in
Explore > SQL Features
or maybe somewhere in Develop
Develop > Best Practices should also have a Batch Nested Loop Join section linking to this page's section

Reply:
Actually I see there is already https://docs.yugabyte.com/stable/explore/ysql-language-features/queries/#join-columns; so this seems more like a Develop page to me

Reply:
Moved into Sql section